Quorum/auto_report/create_report.py

```diff
@@ -14,16 +14,20 @@ def parse_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser(description='This tool generates automatic proposal reports.')
     parser.add_argument('--proposal_id', required=True, type=int, help='The proposal id to generate report to.')
     parser.add_argument('--template', default=DEFAULT_TEMPLATE_PATH, help='The report template to use.')
+    parser.add_argument('--save_to', type=Path, help='Specify where to save the report.')
```

Comment:
Please change naming.
generate_report_path can be an option.

Reply:
done